ui/src/components/layouts/sider/LayoutSider.tsx

```diff
@@ -3,31 +3,71 @@
  * SPDX-FileCopyrightText: Huawei Inc.
  */
 
-import { Image, Menu } from 'antd';
-import Sider from 'antd/es/layout/Sider';
-import { useState } from 'react';
-import { Link } from 'react-router-dom';
-import { homePageRoute, usernameKey } from '../../utils/constants';
+import { Image, Menu, Layout } from 'antd';
+import {useEffect, useState} from 'react';
+import {Link, useNavigate} from 'react-router-dom';
+import {homePageRoute, usernameKey} from '../../utils/constants';
 import registerPanelMenu from '../../content/register/registerPanelMenu';
-import { MenuItemType } from 'antd/lib/menu/hooks/useItems';
+import {ItemType} from "antd/es/menu/hooks/useItems";
+import {serviceVendorApi} from "../../../xpanse-api/xpanseRestApiClient";
+import {AppstoreAddOutlined, BarsOutlined} from "@ant-design/icons";
+
+const mapper: any = {
```

Comment:
better to avoid this additional mapping. Instead a generic method which manipulates the string is better.

Reply:
The categories returned have different formats, so mapper is a little more concise.